### PR TITLE
#260 C++ SDK README

### PR DESCRIPTION
**Pull request by yhmo**
#260 C++ SDK README

### CODE REVIEW

**Review by youny626 — APPROVED** [review]

File: core/src/sdk/README.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+### Build C++ sdk
```
Comment (jielinxu) [inline core/src/sdk/README.md]: SDK

File: core/src/sdk/README.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+The C++ sdk source code is under milvus/core/src/sdk. Build entire milvus project will also build the sdk project.
```
Comment (jielinxu) [inline core/src/sdk/README.md]: The C++ SDK source code is located under milvus/core/src/sdk. It is automatically built when you build the milvus project.

File: core/src/sdk/README.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
+### Build C++ sdk
+
+The C++ sdk source code is under milvus/core/src/sdk. Build entire milvus project will also build the sdk project. 
+If you don't want to build entire milvus project, you can do the following steps:
```
Comment (jielinxu) [inline core/src/sdk/README.md]: If you don't want to build the entire milvus project, follow below steps:

File: core/src/sdk/README.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+ $ ./build.sh -l
+ 
+ # build C++ sdk project
+ $ cd [Milvus root path]/core/cmake_build
+ $ make -C src/sdk
+```
+
+### Try C++ example
+
+Firstly you need to launch a milvus server.
```
Comment (jielinxu) [inline core/src/sdk/README.md]: Firstly, you need to start a Milvus server.

File: core/src/sdk/README.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+If you already build entire milvus project, just run:
```
Comment (jielinxu) [inline core/src/sdk/README.md]: If you've already built the entire milvus project, just start Milvus server with the following command:

File: core/src/sdk/README.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+```shell
+ # start milvus server
+ $ cd [Milvus root path]/core
+ $ ./start_server.sh
+```
+You also can pull milvus release docker image to launch milvus server:
```
Comment (jielinxu) [inline core/src/sdk/README.md]: You can also use Docker to start Milvus server:

File: core/src/sdk/README.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+```shell
+ # run milvus C++ example
+ $ cd [Milvus root path]/core/cmake_build/src/sdk/examples/simple
+ $ ./sdk_simple
+```
+
+### Make your own C++ client project
+
+Firstly create a project folder. And copy C++ sdk header and library files into the folder.
```
Comment (jielinxu) [inline core/src/sdk/README.md]: Create a folder for the project, and copy C++ SDK header and library files into it.

File: core/src/sdk/README.md [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+ $ mkdir MyMilvusClient
+ $ cd MyMilvusClient
+ 
+ # copy necessary files
+ $ cp [Milvus root path]/core/cmake_build/src/sdk/libmilvus_sdk.so .
+ $ cp [Milvus root path]/core/src/sdk/include/MilvusApi.h .
+ $ cp [Milvus root path]/core/src/sdk/include/Status.h .
+```
+
+Create main.cpp under the project folder, and paste the following code into the file:
```
Comment (jielinxu) [inline core/src/sdk/README.md]: Create file `main.cpp` under the project folder, and copy the following codes into it.

File: core/src/sdk/README.md [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+  conn->Connect(param);
+  
+  // put your client code here
+  
+  milvus::Connection::Destroy(conn);
+  return 0;
+}
+```
+
+Create CMakeList.txt under the project folder, and paste the following code into the file:
```
Comment (jielinxu) [inline core/src/sdk/README.md]: Create file `CMakeList.txt` in the project folder, and copy the following codes into it:

**Review by jielinxu — APPROVED** [review]

**Pull request merged**